.pylintrc

```diff
@@ -293,7 +293,7 @@ ignored-classes=optparse.Values,thread._local,_thread._local,Dvcfile
 # (useful for modules/projects where namespaces are manipulated during runtime
 # and thus existing member attributes cannot be deduced by static analysis. It
 # supports qualified module names, as well as Unix pattern matching.
-ignored-modules=
+ignored-modules=azure
```

Comment:
@JordanSimba could you remind me please, why do we need this?

Reply:
I kept getting linting failures for Azure identity module which is imported

Reply:
@skshetry @efiop hey guys, you should know more on these two things ^^ could you help us please?

Reply:
`azure.identity` uses the `azure`'s namespace, though apparently pylint fails to detect this. IMHO it is safe to ignore it here rather than keeping `# pylint disable=no-name-in-module` everywhere.